mlflow/langchain/__init__.py

```diff
@@ -132,9 +132,15 @@ def _infer_signature_from_input_example_for_lc_model(
 ):
     from mlflow.langchain.utils.chat import _ChatResponse
 
-    signature, prediction = _infer_signature_from_input_example(
+    result = _infer_signature_from_input_example(
         input_example, wrapped_model, return_prediction=True, no_conversion=example_no_conversion
     )
+
+    if result is None:
```

Comment:
why not making `_infer_signature_from_input_example` returning `None, None` in this case ? so it is easier for caller handling.

Reply:
If we made that change, `_infer_signature_from_input_example` returns 4 types of objects. Do you think that's simpler?

Reply:
ok make sense